best-practices/client-side/framework/react/README.md

```diff
@@ -20,6 +20,11 @@ To get more familiar with the structure of a React/React Native based project, b
 -   [Jest](https://jestjs.io/) is a JavaScript library for creating, running, and structuring tests.
 -   [Enzyme](https://enzymejs.github.io/enzyme/) created by Airbnb is a JavaScript Testing utility for React.
 
+## Helpful React Development Web Plugins
```

Comment:
*Revised* Day 3 Checklist! Here's a couple of helpful React development web plugins. 👍🏾